backend/src/test/java/codezap/fixture/CategoryFixture.java

```diff
@@ -1,18 +1,25 @@
 package codezap.fixture;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
 import codezap.category.domain.Category;
 import codezap.member.domain.Member;
 
 public class CategoryFixture {
-    public static Category getFirstCategory() {
-        return new Category(1L, MemberFixture.getFirstMember(), "카테고리 없음", true, 1);
+    public static Category getDefaultCategory(Member member) {
+        return new Category(1L, member, "카테고리 없음", true, 1);
     }
 
-    public static Category getSecondCategory() {
-        return new Category(2L, MemberFixture.getSecondMember(), "자바", true, 2);
+    public static Category getCategory(Member member) {
```

Comment:
```suggestion
    public static Category getAdditionalCategory(Member member) {
```
```suggestion
    public static Category getCustomCategory(Member member) {
```
멤버가 가지는 카테고리는 하나가 아니기 때문에 그냥 `getCategory()` 라고 하면 조건에 해당하는 특정 카테고리를 반환해야 할 것 같아서요.
기본 카테고리가 아닌 카테고리를 반환한다는 뉘앙스가 포함되어도 좋을 것 같아요.

Reply:
오 getAdditionalCategory 좋네요 👍